src/blocks/TheaAleksovDev4/script.js

```diff
@@ -0,0 +1,22 @@
+const container = document.querySelector(".container")
+
+function Animal(name, age, contactPhone, image){
+    this.name = name;
+    this.age = age;
+    this.contactPhone = contactPhone;
+    this.image = image
+}
+
+function addAdoption(animalObj){
+    container.innerHTML += `
+    <div class="animal">
+    <img class="image" src=${animalObj.image}>
+    <h1 class="name">${animalObj.name} (${animalObj.age} old)</h1>
+    <h2 class="contact"><i class="fa-solid fa-phone"></i> ${animalObj.contactPhone}</h2>
+    </div class="animal">`
+}
+
+
+addAdoption(new Animal("Lily","3 months","012-345678","./images/1.jpeg"))
```

Comment:
What are the licenses for the images? We need to make sure that we don't violate copyright.

Reply:
They are from Google. Can I replace them with some images from Pexels?

Reply:
Yes, that's a good idea.